src/util/FileSystemException.cpp

```diff
@@ -0,0 +1,16 @@
+// Copyright 2019 Stellar Development Foundation and contributors. Licensed
+// under the Apache License, Version 2.0. See the COPYING file at the root
+// of this distribution or at http://www.apache.org/licenses/LICENSE-2.0
+
+#include "FileSystemException.h"
+
+namespace stellar
+{
+
+FileSystemException::FileSystemException(std::string const& msg)
```

Comment:
why not just having all of this inline and avoid adding a new cpp that doesn't really bring anything

Reply:
good idea, updated